api/ceramic.yaml

```diff
@@ -519,6 +519,62 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/ErrorResponse"
+  "/peers":
+    options:
+      summary: cors
```

Comment:
It'd be nice to add this on all routes by defining it once but I couldn't figure out a way last time I looked. Nothing to do, just putting it in your head in case you know of one or see it sometime 😀